README.md

```diff
@@ -87,5 +87,5 @@ To completely white out the Vagrant machine and build a new one from scratch,
 you can use the following command.
 
 ```
-$ vagrant destroy -f && vangrant ssh
+$ vagrant destroy -f && vangrant up
```

Comment:
Ooops. I don't know where that `ssh` came from. I use `up` all the time myself.

Reply:
This can be your one allowed typo for today

Reply:
Oh no. I'm I allowed to take out a loan on tomorrow's typo?

Reply:
> Oh no. I'm I allowed to take out a loan on tomorrow's typo?

Yes, but you just spent that, too